### PR TITLE
Switch to recommending Node 6

### PR DESCRIPTION
**Pull request by tjvantoll**

### CODE REVIEW

File: start/ns-setup-win.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Complete the following steps to set up NativeScript on your Windows development
 
     1. In the command prompt, run the following command.
 
-        <pre class="add-copy-button"><code class="language-terminal">choco install nodejs -y -version 4.6.0
+        <pre class="add-copy-button"><code class="language-terminal">choco install nodejs -y -version 6.9.1
```
Comment (tjvantoll) [inline start/ns-setup-win.md:51]: Can someone on Windows run this for me to make sure it works as intended? My Windows VM is currently busted.

File: start/quick-setup.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ The NativeScript CLI is built on Node.js, and as such you need to have Node.js i
 You can check whether you have Node.js set up by opening a terminal or command prompt on your development machine and executing `node --version`. If you get an error, head to  <https://nodejs.org/> and download and install the latest “LTS” (long-term support) distribution for your development machine.
 
 > **TIP**:
-> * If you’re on OS X and use [Homebrew](http://brew.sh/), you can alternatively install the Node.js LTS release by running `brew install homebrew/versions/node4-lts` in your terminal.
+> * If you’re on OS X and use [Homebrew](http://brew.sh/), you can alternatively install the Node.js LTS release by running `brew install node6-lts` in your terminal.
```
Comment (jasssonpet) [inline start/quick-setup.md:25]: I think this will revert the changes in https://github.com/NativeScript/docs/pull/563.

Comment (tjvantoll) [reply start/quick-setup.md:25]: Indeed it will. I had to merge that one asap because `brew install node4-lts` no longer works at all. Any concerns with using `brew install node6-lts`?

Comment (jasssonpet) [reply start/quick-setup.md:25]: The same issue will happen on a clean machine:

``` shell
$ brew install node6-lts
Error: No available formula with the name "node6-lts" 
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
This formula was found in a tap:
homebrew/versions/node6-lts
To install it, run:
  brew install homebrew/versions/node6-lts
```

You'll have to do `brew tap versions` (or install a package from there the first time) to be able to only do `brew install node6-lts`.

I would suggest to stick with the `homebrew/versions/node6-lts` part as in the previous PR.

Comment (tjvantoll) [reply start/quick-setup.md:25]: Ah! Ok that makes a lot of sense. Thanks for following up @jasssonpet; I’ll update to `homebrew/versions/node6-lts` before I merge this.

**Comment by tjvantoll** [comment]
Closed with 57ea17626ae1507ed5ef26ccea737811fc0f41b1 and a12eba9506a9756ff1dbfbfa68ff0e1b3c327210.
